index.js

```diff
@@ -87,29 +96,26 @@ const sendResponse = ( req, settings, fn ) => {
 		return fn( wpe, null, headers );
 	} );
 
-	if ( processResponseInStreamMode ) {
-		if ( ! req.xhr ) {
-			throw new Error( 'processResponseInStreamMode is not yet implemented for Node.js' );
-		}
-
+	if ( req.xhr ) {
+		// web
 		req.xhr.addEventListener( 'readystatechange', event => {
 			if ( event.target.readyState !== XMLHttpRequest.HEADERS_RECEIVED ) {
 				return;
 			}
 
-			const type = event.target.getResponseHeader('Content-Type');
-
-			if ( type.split( ';' )[ 0 ] !== 'application/x-ndjson' ) {
-				return;
+			if ( shouldProcessInStreamMode( event.target.getResponseHeader( 'Content-Type' ) ) ) {
+				enableStreamModeProcessing( req, onStreamRecord );
 			}
-
-			enableStreamModeProcessing( req, onStreamRecord );
 		} );
 	}
 
 	return req;
 };
 
+function shouldProcessInStreamMode( contentType ) {
+	return contentType.split( ';' )[ 0 ] === 'application/x-ndjson';
```

Comment:
The `Content-Type` header might be missing in the HTTP response, and then the `contentType` value will be probably undefined or null. Better guard for it before calling `.split` on it.

Another way to implement this is to use a regexp:
```js
return /^application\/x-ndjson($|;)/.test( contentType );
```
Here the `test` method accepts `null` or `undefined` and returns `false` without crashing.

Reply:
done in a048023ab8632cf1e737ca29e5d3205b3d4d85a9